src/Router/index.js

```diff
@@ -33,7 +33,12 @@ const router = createRouter({
             path: "/category/edit/:id",
             component: () => import("../views/CategoryEdit.vue"),
             props :true
-        }
+        },
+        {
+            path: "/category/:name/addtodo",
```

Comment:
What is `:name` in this route? which name?

Reply:
No, this is wrong. No category will be sent to the add todo page. Instead we need to show a dropdown on add todo form with all available categories for that user.

Reply:
okay sir.. i will do changes.